xml/System.Text.Encodings.Web/TextEncoder.xml

```diff
@@ -261,10 +279,10 @@
         <Parameter Name="unicodeScalar" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="unicodeScalar">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="unicodeScalar">A Unicode scalar value.</param>
+        <summary>Determines if a given Unicode scalar value will be encoded.</summary>
+        <returns>Returns <see langword="true" /> if the <paramref name="unicodeScalar"/> value will be encoded by this encoder; otherwise, returns <see langword="false" />.</returns>
```

Comment:
We don't need to include "returns" in the return value description. So, "true if the unicodeScalar value...; otherwise, false.
This applies to all methods with a Boolean return value.